tests/vm_test.go

```diff
@@ -1016,12 +1016,12 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			It("Should force stop a VMI", func() {
```

Comment:
>  Remove newRandomVmi call from vm_test.go and use libvmi instead. 

The commit message would shorter and clearer as `vm_test: replace NewRandomVmi call with libmvi`

Reply:
Done.